themes/default/content/blog/deploy-to-multiple-regions/index.md

```diff
@@ -0,0 +1,759 @@
+    # ...
+    buck:
+        type: aws:s3:Bucket
+        properties:
+            # ...
+        options:
+            provider: ${awsWest2}
+```
+
+{{% /choosable %}}
+
+This overrides the default of using whatever was set at the CLI, and will instead use the programmatically configured settings. Note also that anything you can set at the CLI is available, not just the region. You can see the full settings available in the package's registry documentation for its `Provider` class (for instance, [AWS's]({{< relref "/registry/packages/aws/api-docs/provider" >}}).
+
+## Multi-Region Deployment In Action!
+
+The architecture for our multi-region setup is going to be very simple so we can focus on the essentials. We will provision a single RDS primary database in one region (us-east-1) and backup to a read replica in an entirely different region -- in fact, on an entirely different continent (eu-west-2)!
+
+This is [described in AWS's docs](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_ReadRepl.html#USER_ReadRepl.XRgn) and is depicted in this diagram:
+
+![AWS RDS Read Replica Architecture](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/images/read-replica-cross-region.png)
+
+Now let's dive into some infrastructure as code. There's a bit of preamble before getting to the meat of the example:
+
+{{% chooser language "typescript,python,go,csharp,yaml" / %}}
+
+{{% choosable language typescript %}}
+
+```typescript
+import * as pulumi from "@pulumi/pulumi";
+import * as aws from "@pulumi/aws";
+import * as random from "@pulumi/random";
+
+// Code from below will goes here.
+```
+
+{{% /choosable %}}
+
+{{% choosable language python %}}
+
+```python
+import pulumi
+import pulumi_aws as aws
+import pulumi_random as random
+
+# Code from below goes here.
+```
+
+{{% /choosable %}}
+
+{{% choosable language go %}}
+
+```go
+package main
+
+import (
+	"github.com/pulumi/pulumi-aws/sdk/v5/go/aws"
+	"github.com/pulumi/pulumi-aws/sdk/v5/go/aws/rds"
+	"github.com/pulumi/pulumi-random/sdk/v4/go/random"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi/config"
+)
+
+func main() {
+	pulumi.Run(func(ctx *pulumi.Context) error {
+        // Code from below goes here.
+    })
+}
+```
+
+{{% /choosable %}}
+
+{{% choosable language csharp %}}
+
+```csharp
+using System.Collections.Generic;
+using Pulumi;
+using Aws = Pulumi.Aws;
+using Random = Pulumi.Random;
+
+return await Deployment.RunAsync(() =>
+{
+    // Code from below goes here.
+});
+```
+
+{{% /choosable %}}
+
+{{% choosable language yaml %}}
+
+```yaml
+name: aws-multireg-rds
+runtime: yaml
+description: A multi-region example creating an RDS primary and read-replica in different regions
+
+# Code from below goes here.
+```
+
+{{% /choosable %}}
+
+Next, we'll make the retention period configurable, but default it to 30 days. Note how we can still use standard configuration features _as well_ as explicit providers alongside one another:
```

Comment:
```suggestion
Next, we'll make the retention period configurable, by default it is 30 days. Note how we can still use standard configuration features _as well_ as explicit providers alongside one another:
```

i don't know if i got this change right, but i think `but` is not the word you were looking for...